cmd/gb-vendor/vendor/_testdata/src/github.com/foo/bar/main.go

```diff
@@ -0,0 +1,12 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/quux/flobble"
+	// "bitbucket.org/fwoop/ftang" // commented out, this is deliberate
```

Comment:
Go for it, add more test fixtures, including a _testdata somewhere in there
and adjust the expected result.

On Tue, Jun 2, 2015 at 4:41 PM, Dávid Kaya <notifications@github.com> wrote:

> In cmd/gb-vendor/vendor/_testdata/src/github.com/foo/bar/main.go
> <https://github.com/constabulary/gb/pull/180#discussion_r31496101>:
>
> > @@ -0,0 +1,12 @@
> > +package main
> > +
> > +import (
> > +	"fmt"
> > +
> > +	"github.com/quux/flobble"
> > +	// "bitbucket.org/fwoop/ftang" // commented out, this is deliberate
>
> We could also add something like import flobble "github.com/quux/flobble"
> or import _ "github.com/quux/flobble".
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/constabulary/gb/pull/180/files#r31496101>.
>